### ISSUE TITLE
Why is geocoder needed to use "osgEarth/ImGuiUtils" ?

### ISSUE THREAD

**Issue by remoe** [gwaldron/osgearth#1628]

**Comment by remoe** [comment]
It would be great to remove GLEW dependency. It is to complicated to integrate. But this is a task for OSG?

**Comment by jasonbeverage** [comment]
Hey Remo, the imgui support is still pretty experimental in osgEarth and just used for the imgui example and a couple of other internal projects.  The geocoder is needed to use the geocoder control in ImGuiUtils.  GLEW is necessary b/c we're using the opengl3 integration of imgui which uses GLEW to render the controls.  If we wanted to remove GLEW we'd have to provide a native OSG backend for imgui that uses OSG's extension loading mechanism instead of GLEW.  GLEW is pretty widely used and easy to install on Windows via vcpkg and I'd imagine it's not too big of a hassle on other platforms (at least Linux).

Long story short the imgui integration is mostly just for internal use at the moment but as we start using it more widely we might be more interested in making some changes to the way it works.

**Comment by remoe** [comment]
Thanks Jason. I don't use vcpkg, because they have a lot of issues which doesn't work well with large multiplatform OSG applications. So I build dependencies with own tools. I found the main GLEW repository doesn't compile with CMake. I try to use "glbinding". They have a working CMake setup :) My motivation behind this is to build a osgEarth tool with imgui.

**Comment by remoe** [comment]
Jason, fyi I've wrote a test with "glbinding". It works ;)

**Comment by remoe** [comment]
@jasonbeverage , I hope PM will integrate ImGui fully. It is really awesome to use it with osgEarth!
